scanpy/plotting/_tools/__init__.py

```diff
@@ -633,3 +636,110 @@ def sim(adata, tmax_realization=None, as_heatmap=False, shuffle=False,
                    highlightsX=np.arange(tmax, n_realizations*tmax, tmax),
                    xlabel='index (arbitrary order)')
         utils.savefig_or_show('sim_shuffled', save=save, show=show)
+
+
+def embedding_density(
+        adata: AnnData,
+        basis: str,
+        key: str,
+        group: Optional[str] = None,
+        color_map: Union[Colormap, str] = 'YlOrRd',
+        greysize: Optional[int] = 20,
+        colorsize:  Optional[int] = 40,
+        vmax:  Optional[int] = 1,
+        vmin:  Optional[int] = 0,
+        save: Union[bool, str, None] = None,
+        **kwargs):
+    """Plot the density of cells in an embedding (per condition)
+
+    Plots the gaussian kernel density estimates (over condition) from the
+    `sc.tl.density()` output.
+
+    This function was written by Sophie Tritschler and implemented into
+    Scanpy by Malte Luecken.
+    
+    Parameters
+    ----------
+    adata : :class:`~anndata.AnnData`
+        The annotated data matrix.
+    basis : `str`
+        The embedding over which the density was calculated. This embedded
+        representation should be found in `adata.obsm['X_[basis]']``.
+    key : `str`
+        Name of the `.obs` covariate that contains the density estimates
+    group : `str`, optional (default: `None`)
+        Category of the observed covariate which will be plotted.
+    """
```

Comment:
can you add an example here? 

Something like (I am copying from the `sc.tl.dendrogram` function.
```
    Examples
    --------
    >>> adata = sc.datasets.pbmc68k_reduced()
    >>> sc.tl.dendrogram(adata, 'bulk_labels')
    >>> sc.pl.dendrogram(adata, 'bulk_labels')
```

Reply:
done, thanks for the tip!